frontend-next-migration/src/entities/Clan/model/clanApi.ts

```diff
@@ -1,7 +1,7 @@
 import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
 import { StateSchema } from "@/app/_providers/StoreProvider";
 import { envHelper } from "@/shared/const/envHelper";
-import { GetClanResponse, GetClansResponse, IClan, IClanCreateDto, IClanUpdateDto, ICreateClanResponse } from "@/entities/Clan";
+import { GetClanResponse, GetClansResponse, IClan, IClanCreateDto, IClanUpdateDto, ICreateClanResponse, IJoin } from "@/entities/Clan";
```

Comment:
Sorry, I need to add it to the wiki page documentation. Basically, the problem here is that we shouldn't use absolute import if we are inside the module we are extracting from, we need to change it to relative everywhere, e.g.: "../../". Please fix this in all the places you see it.

Reply:
Noted and code used as example corrected.